stack-7.8.yaml

```diff
@@ -76,6 +76,7 @@ extra-deps:
 - text-metrics-0.1.0
 - pid1-0.1.0.0
 - file-embed-0.0.10
+- yaml-0.8.20
```

Comment:
Doh, I'm confusing myself with these separate PRs, I'd already bumped the LTS versions in a separate one. I'll do that here too.